timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimeLockServer.java

```diff
@@ -151,11 +152,17 @@ public TimeLockServices createInvalidatingTimeLockServices(String client) {
                 ManagedTimestampService.class,
                 createPaxosBackedTimestampService(client),
                 client);
+        LockServerOptions lockServerOptions = new LockServerOptions() {
+            @Override
+            public boolean isSlowLogEnabled() {
+                return true;
```

Comment:
yeah, this is a bit odd.  The slowlog is not configurable on the timelock server...